ivy/functional/backends/torch/experimental/losses.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+import torch
+from ivy.func_wrapper import with_unsupported_dtypes
+from . import backend_version
+
+# Assuming ivy and backend_version are imported and defined properly
+
+
+@with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, backend_version)
+def huber_loss(
+    input: torch.Tensor,
+    target: torch.Tensor,
```

Comment:
Again, the function signature should match. These should be renamed to `true` and `pred`